packages/ts-morph/src/compiler/types/Type.ts

```diff
@@ -233,9 +233,19 @@ export class Type<TType extends ts.Type = ts.Type> {
     }
 
     /**
-     * Gets the target type of a type reference if it exists.
+     * Gets the reference type if this type is an instantiated type.
```

Comment:
Is a "reference type" the same thing as "type reference"? In this case it's getting the target type from a type reference. Here's some docs in the compiler API:

https://github.com/microsoft/TypeScript/blob/4a9e2be386d558cf66d4bd90591c49498419bcf0/lib/typescript.d.ts#L2534

I guess we need different language for what "target type" means. Seems like we could maybe say "Gets the generic type of a type reference."?